data/src/main/java/ca/bc/gov/data/local/entity/VaccineDoseEntity.kt

```diff
@@ -32,5 +33,7 @@ data class VaccineDoseEntity(
     val providerName: String?,
     @ColumnInfo(name = "lot_number")
     val lotNumber: String?,
-    val date: Instant
+    val date: Instant,
+    @ColumnInfo(name = "data_source", defaultValue = "")
```

Comment:
VaccineRecordEntity has the data_source, which is not required here. Please remove the new column.